test/Riok.Mapperly.Tests/Mapping/StaticMethodConversionTest.cs

```diff
@@ -0,0 +1,3006 @@
+using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Tests.Mapping;
+
+public class StaticMethodConversionTest
+{
+    [
+        Theory,
+        InlineData("byte", "class  A { public static A Create(byte source) => new(); }", "return global::A.Create(source);"),
+        InlineData("byte", "struct A { public static A Create(byte source) => new(); }", "return global::A.Create(source);"),
+        InlineData("byte", "class  A { public static A CreateFrom(byte source) => new(); }", "return global::A.CreateFrom(source);"),
+        InlineData("byte", "struct A { public static A CreateFrom(byte source) => new(); }", "return global::A.CreateFrom(source);"),
```

Comment:
This code base usually structures one attribute per attributelist (`[Theory]\n[InlineData]\n...`)